templates/zerver/help/add-custom-emoji.md

```diff
@@ -1,7 +1,8 @@
 # Add custom emoji
 
-If you are an administrator of a Zulip organization, you can add
-custom emojis to your organization for other members to use.
+{!admin-only.md!}
```

Comment:
We actually don't have a doc for enabling this setting... what should I do about this then?